.github/workflows/ci.yml

```diff
@@ -112,6 +113,8 @@ jobs:
         - name: Node.js 19.x
           node-version: "19.7"
 
+        - name: Node.js 20.x
+          node-version: "20.11.1"
```

Comment:
```suggestion
          node-version: "20.11"
```

Reply:
Thanks @aravindvnair99. I reverted that commit